src/include/sof/audio/component.h

```diff
@@ -586,6 +586,9 @@ struct comp_dev {
 	bool is_shared;		/**< indicates whether component is shared
 				  *  across cores
 				  */
+	bool deep_buffering;	/**< indicates whether component is able to work
```

Comment:
difference with long_period?

Reply:
No, but I wanted them to be easily distinguishable in the code.

Reply:
unless code can't referenced, if can keep one flag, I would suggest keep one flag.

Reply:
seems all modified to deep_buffering, what I mean is: possible only use one bool, instead of use two.

Reply:
I misunderstood original comment. I don't see way to replace those two flags with one right now.